informer/disk/disk.go

```diff
@@ -12,13 +12,21 @@ import (
 	"github.com/ipfs/ipfs-cluster/api"
 )
 
+// TODO: switch default to disk-freespace
+const DefaultMetric = "disk-reposize"
+
 var logger = logging.Logger("diskinfo")
 
 // MetricTTL specifies how long our reported metric is valid in seconds.
 var MetricTTL = 30
 
 // MetricName specifies the name of our metric
-var MetricName = "disk"
+var MetricName string
```

Comment:
Ah, okay I see. I was a bit uncertain if I should change that or not, your explanation makes sense though.